### ISSUE TITLE
error when content-length = 0

### ISSUE THREAD

**Issue by mastef** [yupswing/akifox-asynchttp#40]
While testing something, I noticed that requests started erroring out with "Blocked" for some reason.

After digging in, traced it down to 
```hx
      contentLength = Std.parseInt(headers.get('content-length'));
...
      if (contentLength > 0) mode = HttpTransferMode.FIXED;
```

So although the response had a `content-length` of `0` correctly defined in the header, it would proceed to `HttpTransferMode.UNDEFINED` which would error out. This is for example the case if you provide the `print=silent` flag to Firebase's REST API ( e.g. you just want to push data, you don't want to get it returned back to you ).

I guess the initial condition is a nice way to catch an undefined/malformed header response, where any string would `parseInt` then to `0`. In my case I added a second check to catch a real `content-length: 0` response : 

```hx
if (headers.get('content-length') == "0") mode = HttpTransferMode.FIXED;
```

**Comment by yupswing** [comment]
if you would like to make a pull request I would be happy to merge it.
otherwise I will make the same change on the master as soon as possibile.
thanks for the fix.

**Comment by mastef** [comment]
Ok, PR sent - good way of finding out that there was an Android crash fixed since 0.4.4, thx :)

**Comment by yupswing** [comment]
Merged in dev branch 487b7fd
It will ship in the next master release
